### PR TITLE
Features/feature 687 messageflow2a

### PR DESCRIPTION
**Pull request by rallen2010**
-- adds support for EE feature advanced message Flow )message start event, message End Event, & message Boundary Event

### CODE REVIEW

File: src/ddl/install_scratch.sql [hunk 1]
```diff
@@ -290,12 +290,26 @@ create table flow_message_subscriptions (
...
+comment on column flow_message_subscriptions.msub_key_value         is 'Message Key Value that must be supplied with intermediate message';
+comment on column flow_message_subscriptions.msub_prcs_id           is 'Process ID to call back for intermediate message';
+comment on column flow_message_subscriptions.msub_sbfl_id           is 'Subflow ID to call back for intermediate message';
+comment on column flow_message_subscriptions.msub_step_key          is 'Step Key  to call back for intermediate message';
+comment on column flow_message_subscriptions.msub_dgrm_id           is 'Diagram ID to use for message start event';
+comment on column flow_message_subscriptions.msub_callback          is 'routine to use for callback';
+comment on column flow_message_subscriptions.msub_callback_par      is 'parameter to use on callback';
+comment on column flow_message_subscriptions.msub_payload_var       is 'process variable to return payload variable into';
+comment on column flow_message_subscriptions.msub_created           is 'timestamp of subscription creation';
```
Comment (commi235) [inline src/ddl/install_scratch.sql:311]: shouldn't the comments be in a separate file now?

Comment (rallen2010) [reply src/ddl/install_scratch.sql:311]: Good catch.  Don't pick that up in the merge. Very old branch!

Comment (commi235) [reply src/ddl/install_scratch.sql:311]: Could you modify the file as needed and resubmit?

Comment (rallen2010) [reply src/ddl/install_scratch.sql:311]: removed

Comment (rallen2010) [reply src/ddl/install_scratch.sql]: removed and added to correct file.   also fixed view (added column) and added comments for view to install_ddl_comments

**Review by commi235 — APPROVED** [review]

**Pull request merged**